js/src/hooks/useLayout.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { useEffect } from '@wordpress/element';
+
+// gla-* styles are defined in .~/css/shared/_woocommerce-admin.scss
+const classNameDict = {
+	'full-page': [
+		'woocommerce-admin-full-screen',
+		'is-wp-toolbar-disabled',
+		'gla-full-page',
+	],
+
+	'full-content': [ 'gla-full-content' ],
```

Comment:
💅  There are several places that are using `full-page` and `full-content`, might be better to define them as constant variables to avoid typo.

Reply:
I have thought about whether to add constants during development. Although the same values are used in many places, the values are only evaluated in `useLayout` hook and are not spilled out to other scopes.

Therefore, I chose to enumerate the valid values by JSDoc, which in some manner should be helpful for avoiding typos.

![Kapture 2022-01-20 at 19 44 01](https://user-images.githubusercontent.com/17420811/150333933-c6334ba1-8cb8-4b76-888b-4fdd07ac6924.gif)

(My editor Sublime Text does not use autocomplete like VSCode, though.) 🙈

Reply:
It makes sense to me, thanks for clarifying.